components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -178,17 +179,47 @@ const facilityHasUnsavedChanges = () => {
         !== JSON.stringify(facilitySections.healthcareProfessionalIds)
         || facilitySections.healthProfessionalsRelations.length
 
-    return areThereUnsavedChanges
+        return areThereUnsavedChanges
```

Comment:
Since the value of this variable is a bool, a simpler name would be `hasUnsavedChanges`.

Reply:
After reviewing this some more I realized that you are declaring `areThereUnsavedChanges` twice in the same function, but they represent very different values. One is if the facility has unsaved changes, and the other is if the healthcare professional has unsaved changes. These should be two distinct variables. Something like `facilityHasUnsavedChanges` and `healthcareProfessionalHasUnsavedChanges`.

Reply:
I thought about this a bit more, and I think it would be better if you kept them as separate functions. Meaning, keep `facilityHasUnsavedChanges` and create a new `healthcareProfessionalHasChanges`. Then call the two functions in `hasUnsavedChanges`. The reason for this is because otherwise it becomes a big function performing multiple things, when we should try to aim for 1 purpose per function.